src/components/Newsletter/Newsletter.js

```diff
@@ -114,21 +125,20 @@ class Newsletter extends React.Component {
             id="fieldjrdtwy"
             maxlength="200"
             name="cm-f-jrdtwy"
-            onChange={value => {
-              console.log(value);
-            }}
+            onChange={value => {}}
```

Comment:
presumably we can just remove the onChange handler now?

Reply:
mm no react complains if we take it out because it's a readonly input